tools/build_api.py

```diff
@@ -459,7 +459,7 @@ def merge_region_list(region_list, destination, notify, padding=b'\xFF'):
     notify.info("Space used after regions merged: 0x%x" %
                 (merged.maxaddr() - merged.minaddr() + 1))
     with open(destination, "wb+") as output:
-        merged.tofile(output, format=format.strip("."))
+        merged.tofile(destination, format=format.strip("."))
```

Comment:
This block doesn't make much sense. Did you mean to remove the `open()` call above? Currently, `output` isn't being used.

Reply:
Ok no worries, sounds like removing the `with open...` statement is what you're after then.